### PR TITLE
Feat/final header things

### PR DESCRIPTION
**Pull request by sdmisra**
# PR Template

- [x] I have ran all tests (if applicable) and they are passing. 

### Types of changes made?

- [x]  testing
- [x]  javascript
- [x]  functionality

### What does this PR do?

- adjusts the header animation to make it a bit smoother / faster
- moves the explore button to below the logo in the middle of the header
- conditional logic for when the explore button appears
- removes discover button from header until that function can be moved into rootSlice (after MVP)
- adds discover functionality! only accessible from homepage though

Added one test for checking for header in welcome.cy.js

### CODE REVIEW

File: src/Components/WelcomePage/WelcomePage.js [hunk 1]
```diff
@@ -15,17 +15,12 @@ export const WelcomePage = () => {
     .then(data=> {
       const addOne = [...displayedWorlds, data]
       dispatch(getDiscoveredWorlds(addOne))
-      // console.log(data.id)
       history.push(`/world/${data.id}`)
     }).catch((error)=>{
       console.log('This line replaced by error handling')
     })
```
Comment (priscillaapaxton) [inline src/Components/WelcomePage/WelcomePage.js:23]: here's something I noticed! Let's get rid of this console log

**Review by priscillaapaxton — COMMENTED** [review]
I think after this console.log is removed, we should be all done!

File: src/Components/WelcomePage/WelcomePage.js (hunk 1 above)
Comment (sdmisra) [reply src/Components/WelcomePage/WelcomePage.js:23]: You got it, pushing up changes now - i both removed that console log and I added error handling on that line 20 to actually properly use your nifty error stuff

**Review by sdtrachsel — APPROVED** [review]
I fixed the merge confilicts. It does appear that the Create in the header doesn't function. I'm going to merge and pull down and see if I can get it fixed.

**Pull request merged**